### PR TITLE
feat: Added AppendSpan

### PR DESCRIPTION
**Pull request by bCamba**
For issue #208

### CODE REVIEW

File: src/LinkDotNet.StringBuilder/ValueStringBuilder.Append.cs [hunk 1]
```diff
@@ -136,6 +136,19 @@ public void AppendLine(scoped ReadOnlySpan<char> str)
         Append(string.Concat(str, Environment.NewLine));
     }
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public Span<char> AppendSpan(int length)
```
Comment (linkdotnet) [inline src/LinkDotNet.StringBuilder/ValueStringBuilder.Append.cs:140]: Can you add an xmldoc?

**Comment by linkdotnet** [comment]
LGTM.

Can you add an entry in the [`CHANGELOG.md`](https://github.com/linkdotnet/StringBuilder/blob/main/CHANGELOG.md).
This file is used for making release notes. You can add a new `### Added` under `UNRELEASED`

**Comment by bCamba** [comment]
Done. Let me know if anything else is needed

**Comment by linkdotnet** [comment]
> Done. Let me know if anything else is needed

Perfect! Thanks for your work. I will directly merge it and create a release

**Pull request merged**